Add blog post: Getting Started with Solidus Starter Frontend

gsmendoza opened the pull request.

gsmendoza commented on source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:
```diff
@@ -0,0 +1,122 @@
+date: 2021-11-25
+tags: frontend
+author: George Mendoza
+cover_image: /blog/2021/11/25/getting-started-with-solidus-starter-frontend/home.png
```
@kennyadsl I may need some help to make this cover image nicer:

![image](https://user-images.githubusercontent.com/61476/144563696-89b77ee9-0410-46b0-8298-f4a96ecd5ef2.png)

kennyadsl replied:
@mfrecchiami ping 🙂

mfrecchiami replied:
@gsmendoza added to my ToDo list, I'll tackle it tomorrow!

gsmendoza commented on source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:
```diff
@@ -0,0 +1,122 @@
+<div class="carousel">
+  <div class="carousel-list">
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/10-home.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Home page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/20-product.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Product page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/30-cart.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Cart page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/40-login.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Login page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/50-checkout-address.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Address page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/60-checkout-delivery.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Delivery page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/70-checkout-payment.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Payment page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/80-checkout-confirm.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Confirm page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/90-checkout-complete.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Complete page</figcaption>
+    </figure>
+  </div>
+</div>
```
@kennyadsl We can customize this carousel to better present the screenshots. Let me know which format in https://kenwheeler.github.io/slick/#demos is ideal.

kennyadsl commented on source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:
```diff
@@ -0,0 +1,122 @@
+
+* Mailers with their corresponding views
+* [View components](https://viewcomponent.org) for complex presentational logic
+* Unit and integrated specs for testing the frontend
```
`integrated` -> `integration`?

kennyadsl reviewed (commented):
Thanks, this is nice! 

I have the feeling we could improve this with some notes about the next steps for the starter frontend in our plans. Can we add a final paragraph about this please?

gsmendoza commented:
@kennyadsl Appreciate if you can review again. Changes:

1. Updated the date stamp
2. Added link to screencast
3. Added next steps for the starter frontend in our plans
4. Minor changes to the content to make them read better

kennyadsl reviewed (approved):
Thanks @gsmendoza! 


@mfrecchiami I think this may be better with a proper cover image, instead of the plain screenshot. Do you have time to create one for us? If you don't in the next days, no problem, just let us know and we'll merge this as is and we can add a better cover later. 🙏

mfrecchiami commented:
@Alberto Vena ***@***.***> I'm OOO until Monday! I can design
a cover the next Tuesday.

Il ven 17 giu 2022, 11:23 Alberto Vena ***@***.***> ha
scritto:

> ***@***.**** approved this pull request.
>
> Thanks @gsmendoza <https://github.com/gsmendoza>!
>
> @mfrecchiami <https://github.com/mfrecchiami> I think this may be better
> with a proper cover image, instead of the plain screenshot. Do you have
> time to create one for us? If you don't in the next days, no problem, just
> let us know and we'll merge this as is and we can add a better cover later.
> 🙏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/solidusio/solidus-site/pull/496#pullrequestreview-1010247686>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALAMOOI45DJSYQZZ2OJKAWTVPQ723ANCNFSM5JJA5AOA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

kennyadsl commented on source/blog/2022-06-16-getting-started-with-solidus-starter-frontend.html.markdown:
```diff
@@ -0,0 +1,134 @@
+SolidusFrontend from the solidus meta-package gem and move its code to a new
+repo. If your app depends on SolidusFrontend, you'll need to explicitly add
+`solidus_frontend` to your Gemfile in order to continue using it.
+
```
Last thing while we wait for the cover: we could add that we are still working on a solution to make the extensions compatible with the new frontend.

gsmendoza commented:
@mfrecchiami Thank you for the cover image!

@kennyadsl I have also added a short note about making the extensions compatible with Starter Frontend. Appreciate if you can take a look!
